src/ai/backend/test/tester/dependency.py

```diff
@@ -80,28 +98,25 @@ class SSEDep(BaseDependencyModel):
 class ClusterDep(BaseDependencyModel):
     # By default, testing is conducted for both single-node and multi-node setups through parametrization,
     # But we'd like to have left room for manually injecting values.
-    cluster_mode: Optional[str] = Field(
-        default=None,
+    cluster_mode: str = Field(
         description="The cluster mode for the session.",
-        examples=["single_node", "multi_node"],
```

Comment:
Let's define it as a StrEnum. Or, if ClusterMode is already in common, it seems like we could use the strenum in common.